ControlRoomApplication/ControlRoomApplication/Controllers/SpectraCyberControllers/AbstractSpectraCyberController.cs

```diff
@@ -474,7 +474,7 @@ protected SpectraCyberRequest GenerateCurrentDataRequest(bool waitForReply = tru
             }
 
             return new SpectraCyberRequest(
-                SpectraCyberCommandTypeEnum.DATA_REQUEST,
+                SpectraCyberCommandTypeEnum.GENERAL_COMMUNICATION,
```

Comment:
I didn't realize this was already a method created, this may cause some unforeseen issue that we aren't accounting for with the GENERAL_COMMUNICATION value since this has lower priority, if it isn't used anywhere else as mentioned then this is probably okay to go.